src/index.html

```diff
@@ -7,12 +7,67 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="image"
+        src="./images/imac.jpeg"
+        alt="imac27"
+      />
+
+      <div class="product">
+        <h1 class="product__name product__name--position">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </h1>
+        <h2 class="product__code product__code--position">
+          Product code: 195434
+        </h2>
+      </div>
+      <div>
+        <ul class="stars stars--position">
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <h2 class="stars__review stars__review--position">Reviews: 5</h2>
```

Comment:
The `<h2>` element should not be a child of the `<ul>` element. Consider placing it outside the `<ul>` to maintain proper HTML semantics.